book.rb

```diff
@@ -1,10 +1,10 @@
 require './item'
 
-class Book < Item 
```

Comment:
[OPTIONAL]: Try not to change variable names(like published_date for publish_date) for modules that we are all going to use without checking with the team. As it is right now, these changes will break my existing Author feature for a name change that will not impact other functionalities. 

This is something I can change for that feature as it is easier than changing your update, but please be mindful of these changes in the future.